server/client.go

```diff
@@ -224,7 +222,7 @@ func (c *Client) WorldItemAdded(id game.EntityId, worldItem *game.WorldItem) {
 }
 
 func (c *Client) WorldItemUpdated(id game.EntityId, worldItem *game.WorldItem) {
-	c.SendLossy(&MsgEntityState{
+	c.Send(&MsgEntityState{
```

Comment:
Wanted to test it. But, we still generate a lot of these messages quickly. So in one loop, we blow through the lossy queue with the updates. Looks bad.